app/controllers/suggestion_controller.rb

```diff
@@ -0,0 +1,75 @@
+# Route for release2
+get "/" do
+  erb :index
+end
+
+# Route for release1
+# get "/" do
+#   redirect "/suggestions"
+# end
+
+
+get "/suggestions" do
+  @suggestions = Suggestion.all
+  erb :"/suggestions/index"
+end
+
+get "/suggestions/:id" do
+  @suggestion = Suggestion.find_by(id: params[:id])
+  puts @suggestion
+  if @suggestion
+    erb :"/suggestions/show"
+  else
+    "Suggestion not found"
+  end
+end
+
+post "/suggestions" do
+  @suggestion = Suggestion.new(title: params[:suggestion][:title],
```

Comment:
1. Look into utilizing namespacing in your form, so you could nest suggestion title, description, user to be like this

```
{ 
  suggestion: {
    title: '',
    description: '',
    user_id: 1
  }
}
```
You could then pass `Suggestion.new(params[:suggestion])

2. The variable you defined also doesn't have to be an instance variable.